### PR TITLE
Enhancements to README and test_all.py

### PR DESCRIPTION
**Pull request by eshaanpathak**
1. Added a check to `src/test_all.py` to ensure that the `.json` file extension ending is not present in `tasks/README.md`. I then went through and removed these extensions so they would pass this test.
2. Fixed the tasks to be in numerical order in `tasks/README.md`.
3. Edited grammar in the description of each task in `tasks/README.md`.
4. Added a check to make sure all characters are lowercased in the filename for consistency throughout filenames. I then went through and lowercased all task filenames so they would pass this test. 
5. There appears to be different tasks under the same task numbers for tasks 119 and 184. You can fix this by renaming them to the last 2 tasks after task submissions are no longer accepted.
6. There are also a lot of gaps between task numbers. Are you going to go by hand and renumber everything after task submissions are no longer accepted?

### CODE REVIEW

File: src/test_all.py [hunk 1]
```diff
@@ -64,19 +64,26 @@
...
+            true_file = file.replace(".json", "")
+            for char in true_file:
+                if char.isupper():
+                    raise Exception(f" * Looks like there is an uppercase letter in `{true_file}`. All letters should be lowercased.")
+
+            if file in task_readme_content:
+                raise Exception(f" * Looks like the .json file extension ending is present with the task name in `tasks/README.md` when it should just be `{true_file}`")
+
+            if true_file not in task_readme_content:
+                raise Exception(f' * Looks like the task name `{true_file}` is not included '
```
Comment (danyaljj) [inline src/test_all.py:86]: Cool! 👏 👏

**Comment by danyaljj** [comment]
Thanks, @eshaanpathak ! This is super helpful! 

> There appears to be different tasks under the same task numbers for tasks 119 and 184. You can fix this by renaming them to the last 2 tasks after task submissions are no longer accepted.
> There are also a lot of gaps between task numbers. Are you going to go by hand and renumber everything after task submissions are no longer accepted? 

Yeah, I think we should sort them out later when things cool down.

**Review by danyaljj — APPROVED** [review]

**Comment by danyaljj** [comment]
Leaving it here for @swarooprm to check before we merge it.

**Review by swarooprm — APPROVED** [review]
LGTM! Thanks @eshaanpathak. This is super-helpful.

**Pull request merged**